editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper.spec.browser.tsx

```diff
@@ -48,7 +48,7 @@ export var App = (props) => {
       }}
     >
       <div data-uid="inner-div">
-        <Card data-uid="card-instance" />
+        <Card data-uid="other-card-instance" />
```

Comment:
why the rename?

Reply:
Because it clashes with the uid in one of the other files (`/src/app.js` from memory), so the LOAD action fails.